qemu/tests/cfg/cpu_device_hotplug_maximum.cfg

```diff
@@ -48,6 +49,9 @@
             vcpu_threads = 8
     variants:
         - @default:
+            ovmf:
+                no Host_RHEL.m8.u4
+                required_qemu = [6.0.0-16, )
         - with_hugepages:
             hugepage = yes
```

Comment:
Could you explain the reason that no hotunplug version limited when use `hugepage = yes` ? 
Is vcpu unplug with hugepage not supported yet ? Thanks!

Reply:
> Could you explain the reason that no hotunplug version limited when use `hugepage = yes` ? Is vcpu unplug with hugepage not supported yet ? Thanks!

Sorry for that I miss the case, update and thanks.